src/logger/simple.py

```diff
@@ -34,7 +35,7 @@ def log_train(self, losses: Optional[dict[str, float]] = None, images: Optional[
             for loss_name, loss_value in losses.items():
                 self.loss_buff['values'][loss_name] += [loss_value]
                 self.loss_buff['sumlast'][loss_name] += loss_value
-                self.wandb.log({f"{loss_name} loss": loss_value})
+                self.wandb.log({f"{loss_name} loss": loss_value})  # type: ignore
```

Comment:
What does this comment mean?

Reply:
We initialize `self.wandb` as `wandb.init(...)` which has the return type `Run | RunDisabled | None` and this comment was to disable mypy warnings that `None` has no attribute `log`. Fixed it by asserting that what we got isn't `None`.